lib/components/home/artwork_rail.js

```diff
@@ -34,7 +34,7 @@ class ArtworkRail extends React.Component {
     if (!this.state.expanded) {
       return (
         <TouchableHighlight style={styles.expansionButton} onPress={this.onPress.bind(this) } underlayColor={'white'}>
-            <View style={{height: 10, width: 10, alignSelf: 'center'}} />
+            <Image style={{height: 8, width: 15, alignSelf: 'center'}} source={require('../../../images/chevron.png')} />
         </TouchableHighlight>
```

Comment:
so the pattern is that we use `Image` for local images, and `ImageView` for network'd ones?